### PR TITLE
Hotfix: Datacube WCS fix + adopt East-West convention

### PR DESCRIPTION
**Pull request by rcooke-ast**
Fixed a long-term bug associated with the WCS. With the help of some data collected by @RaymondRemigio, we identified the bug, which was due to incorrectly evaluating the WCS. With this PR, we also update the WCS to adopt the East/West convention of East to the left.

Below is a series of 8 white light images with their WCS locked to the same value. The green circle is the "true" value of the star coordinate. Regardless of the position angle, the WCS of all frames is close to equal. The remaining offsets is possibly the result of not centering the star on REFA and base was not marked during the observations.

image

### CODE REVIEW

**Review by kbwestfall — APPROVED** [review]
Looks good to me.

File: pypeit/core/datacube.py [hunk 1]
```diff
@@ -979,7 +979,7 @@ def create_wcs(raImg, decImg, waveImg, slitid_img_gpm, dspat, dwave,
 
     # Generate a master WCS to register all frames
     coord_min = [_ra_min, _dec_min, _wave_min]
-    coord_dlt = [dspat, dspat, dwave]
+    coord_dlt = [-dspat, dspat, dwave]
```
Comment (profxj) [inline pypeit/core/datacube.py:982]: ah, a negative sign.. :)

**Review by profxj — APPROVED** [review]
thanks for this!

**Comment by rcooke-ast** [comment]
Thanks! Merging...

**Pull request merged**